src/caikit_nlp_client/__init__.py

```diff
@@ -0,0 +1,5 @@
+# noqa: F401
+from .grpc_client_introspection import GrpcClient, GrpcConfig
+from .http_client import HttpClient, HttpConfig
+
+__all__ = ["GrpcConfig", "GrpcClient", "HttpClient", "HttpConfig"]
```

Comment:
running `from caikit_nlp_client import *` will bring in all of these

Reply:
cool